dev/run-tests.py

```diff
@@ -506,6 +506,8 @@ def main():
         test_env = "amplab_jenkins"
         # add path for Python3 in Jenkins if we're calling from a Jenkins machine
         os.environ["PATH"] = "/home/anaconda/envs/py3k/bin:" + os.environ.get("PATH")
+        # add java 7 to the environment too
+        os.environ["JAVA_7_HOME"] = "/usr/java/latest"
```

Comment:
If you make the changes to jenkins I can remove it; otherwise I had no way of testing this patch on jenkins.